src/action/log.js

```diff
@@ -36,7 +37,13 @@ export function error(...args) {
   pushLogs(''); // newline
   pushLogs(`ERROR: ${args[0]}`);
   for (let i = 1; i < args.length; i++) {
-    pushLogs(JSON.stringify(args[i], null, '    '));
+    pushLogs(
+      JSON.stringify(
```

Comment:
Is this reason for this that on mobile, errors are already objects? Why not always print the `stringify`'d object?

Reply:
Error objects have different formating on mobile and desktop. Since the grcp layer throws difference types. Just making sure we print the most readable version here.

Reply:
Ahh okay. LGTM then!